lib/api/bucketDelete.js

```diff
@@ -20,6 +21,8 @@ function bucketDelete(authInfo, request, log, cb) {
 
     if (authInfo.isRequesterPublicUser()) {
         log.debug('operation not available for public user');
+        monitoring.promMetrics(
+            'DELETE', request.bucketName, 403, 'deleteBucket');
```

Comment:
Instead of hardcoding the error codes can you use `err.code` instead?

Reply:
Not everywhere we  cb with "err" so I used https://github.com/scality/Arsenal/blob/063361377cbc3050fcd6eb94b18be8e526ab17ab/errors/arsenalErrors.json
to get the error codes.